project/Dependencies.scala

```diff
@@ -39,7 +39,9 @@ object Dependencies {
   )
 
   val javaxMail = Seq(
-    "com.sun.mail" % "jakarta.mail" % javaxMailVersion
+    "com.sun.mail" % "imap"  % javaxMailVersion,
+    "com.sun.mail" % "smtp"  % javaxMailVersion,
+    "com.sun.mail" % "gimap" % javaxMailVersion
```

Comment:
Interesting, are these available for some time? I must have missed it back then. I guess the first two were bundled before in one jar, the last is a new dependency?

Reply:
Yes, exactly, the classes in `gimap` were not in the "fat jar" `jakarta.mail`.